tobac/utils.py

```diff
@@ -91,47 +125,91 @@ def mask_cube_untracked(variable_cube, mask):
 
 
 def mask_cube(cube_in, mask):
-    """Mask cube where mask is larger than zero
-    Input:
-    cube_in:           iris.cube.Cube
-                       unmasked data cube
-    mask:              numpy.ndarray or dask.array
-                       mask to use for masking, >0 where cube is supposed to be masked
-    Output:
-    cube_out:          iris.cube.Cube
-                       Masked cube
+    """Mask cube where mask (array) is not zero.
+
+    Parameters
+    ----------
+    cube_in : iris.cube.Cube
+        Unmasked data cube.
+
+    mask : numpy.ndarray or dask.array
```

Comment:
does this function really work when mask is given as `np.ndarray()?` I think `np.ndarray` has no attribute `core_data()` or am I missing something here?

Reply:
No, this wouldn't work. The input here has to be a iris.cube.Cube, like for all the other masking functions. I don't know how this slipped in here, thanks for noticing this!